service/service.go

```diff
@@ -134,3 +140,30 @@ func (s *Service) Remove(id []byte) error {
 
 	return s.db.Delete(id)
 }
+
+func GetObject(id []byte) (*GetObjectResult, error) {
+	return s.GetObject(id)
+}
+
+func (s *Service) GetObject(id []byte) (*GetObjectResult, error) {
+	in, err := s.db.GetVal(id)
+	if err != nil {
+		return nil, err
+	}
+	if in <= 0 {
+		return nil, fmt.Errorf("Id(%s) is not in DB", id)
+	}
+	vector, err := gongt.GetStrictVector(in)
```

Comment:
if you need float64 vector use [gongt.GetVector](https://github.com/yahoojapan/gongt/blob/master/gongt.go#L775) function instead of gongt.GetStrictVector

But raw vector is []float32 I think we don't need float64 vector for json or gRPC, so can you fix it to float32 and use  gongt.GetStrictVector

Reply:
thank you for your review.
I have no problem to change vector type to []float32 too.